lib/motions/general-motions.coffee

```diff
@@ -150,6 +150,7 @@ class MoveRight extends Motion
   moveCursor: (cursor, count=1) ->
     _.times count, =>
       cursor.moveRight() unless cursor.isAtEndOfLine()
+      cursor.moveRight() if atom.config.get('vim-mode.wrapLeftRightMotion') and cursor.isAtEndOfLine()
```

Comment:
Does `l` still work with this change? It used to call `cursor.moveRight()` *unless* the cursor was at the end of the line. Now, if `wrapLeftRightMotion` is `true`, it does it *only if* the cursor is at the end of the line.

I would think it should be this:

```coffee
cursor.moveRight() if (not cursor.isAtEndOfLine()) or atom.config.get('vim-mode.wrapLeftRightMotion')
```